platform/ui/src/components/MeasurementTable/MeasurementTable.tsx

```diff
@@ -36,6 +36,8 @@ const MeasurementTable = ({ data, title, onClick, onEdit, servicesManager }) =>
               item={measurementItem}
               onClick={onClick}
               onEdit={onEdit}
+              measurementService={measurementService}
+              commandsManager={commandsManager}
```

Comment:
So that we can pass that to the custom MeasurementItem's and then they can have their own functions that use these services. For instance, one could create a custom MeasurementItem that has a context menu, which is not necessarily something that you want to do for the open source.

What is the alternative in a case like that, where it's not "onClick" or "onEdit"? What if they want to have N other custom functions inside the MeasurementItem?

Reply:
I see CustomMeasurementItem is coming from customizationService, so theoretically you can bind these services when you register them in the customization service no?

Reply:
You're right. I'll close this PR. Thanks.